packages/tc-markdown-parser/__tests__/nested-property-value.spec.js

```diff
@@ -8,14 +8,15 @@ schema.init({
 });
 
 const parser = getParser({
-	type: 'MainType',
+	type: 'RelationshipTypeOne',
 });
 
 // The weird string concatenation in the markdown fixture is so that
 // dev tooling doesn't tidy up the 'extraneous' whitespace
 const addLineBreaks = str => str.replace(/\t+/g, '\t').split('\n').join('  \n');
 
-describe('nested property definition tests', () => {
+// we are soon to remove this functionality so not worried about updating tests to new schema
```

Comment:
Highlight #10 We'll soon be removing support for relationships in runbook.md so not bothering to refactor these tests to use the new types